packages/@react-aria/menu/src/useMenuTrigger.ts

```diff
@@ -70,7 +70,7 @@ export function useMenuTrigger(props: MenuTriggerProps, state: MenuTriggerState)
     menuTriggerProps: {
       ...triggerAriaProps,
       id: menuTriggerId,
-      onPress,
+      onPressStart: onPress,
```

Comment:
I don't follow this change, can you explain?

Reply:
see PR description.